themes/bootstrap3/less/components/offcanvas.less

```diff
@@ -23,17 +23,12 @@
   }
 
   .offcanvas .close-offcanvas { display: block; }
-  .search-filter-toggle {
-    &:extend(.btn);
-    &:extend(.btn-default);
-    margin-top: 1rem;
-    margin-bottom: 1rem;
-  }
```

Comment:
Are we simply removing this styling?
Are we going to replace this with another style?
Should we add a replacement to the Gruntfile make this automatic?

Reply:
@crhallberg, it sounds like @ckaz felt these styles were redundant and are actually defined elsewhere -- see his notes in the main comment on the PR for more details. (And don't feel bad -- I had the same questions and missed the notes on my first read-through too :-) )

Reply:
The JIRA description seemed to me like he was going to move it outside of a media query to fix the error. 

> Since .search-filter-toggle is only shown on XS devices anyway, we could [use] extend outside the media query...

Assumed use inserted. It seems here that he just removed it.

Reply:
Above, he says:

"    * Since the class that is extended in the media query is used on small devices only as defined in the PHTML templates, we can extend outside the media query and should get the same results. In fact, this already happens in lines 5--10. So I just deleted the duplicated content inside the media query and we should be fine? Or am I mistaken?"

Does that explain it?

Reply:
Sorry for the delay and for being confusion: Originally I had taken the approach of moving the @extend-code outside the media query. Then I discovered, that it was already declared there in full (lines 5--10). Therefore it was just deleted inside the media query.
Does that explain it?

Reply:
Don't know how I missed this. Sorry for the confusion!

Reply:
It's easy to miss. No worry. Cheers!